setup.py

```diff
@@ -30,6 +30,9 @@
         'console_scripts': ['openfisca-run-test=openfisca_core.scripts.run_test:main'],
         },
     extras_require = {
+        'parsers': [
+            'OpenFisca-Parsers >= 1.0.1, < 2.0',
```

Comment:
That doesn't seem right : `parsers#1.0.1` depends on `OpenFisca-Core >= 4.0.0b1, < 6.0`. This will lead us to a version conflict.

We should merge [this PR](https://github.com/openfisca/openfisca-parsers/pull/27/files) before, and declare as extra require `'OpenFisca-Parsers >= 1.0.2, < 2.0'`.